packages/blob-storage-manager/src/env.ts

```diff
@@ -1,59 +1,24 @@
-import { z } from "zod";
-
-function transformToBoolean(varName: string) {
-  return (arg: string, ctx: z.RefinementCtx): boolean => {
-    const arg_ = arg.toLowerCase();
-    if (arg_ !== "true" && arg_ !== "false") {
-      ctx.addIssue({
-        code: z.ZodIssueCode.custom,
-        message: `${varName} must be either true or false`,
-      });
-
-      return z.NEVER;
-    }
-
-    return arg_ === "true";
-  };
-}
-
-const envSchema = z.object({
-  BEE_DEBUG_ENDPOINT: z.string().url().optional(),
-  BEE_ENDPOINT: z.string().url().optional(),
-  CHAIN_ID: z
-    .string()
-    .min(1)
-    .default("7011893055")
-    .transform((value, ctx) => {
-      const chainId = parseInt(value, 10);
-
-      if (isNaN(chainId) || chainId <= 0) {
-        ctx.addIssue({
-          code: z.ZodIssueCode.custom,
-          message: `CHAIN_ID is invalid: ${chainId}`,
-        });
-
-        return z.NEVER;
-      }
-
-      return chainId;
-    }),
-  GOOGLE_STORAGE_BUCKET_NAME: z.string().optional(),
-  GOOGLE_STORAGE_PROJECT_ID: z.string().optional(),
-  GOOGLE_SERVICE_KEY: z.string().optional(),
-  GOOGLE_STORAGE_API_ENDPOINT: z.string().url().optional(),
-
-  GOOGLE_STORAGE_ENABLED: z
-    .string()
-    .default("false")
-    .transform(transformToBoolean("GOOGLE_STORAGE_ENABLED")),
-  POSTGRES_STORAGE_ENABLED: z
-    .string()
-    .default("false")
-    .transform(transformToBoolean("POSTGRES_STORAGE_ENABLED")),
-  SWARM_STORAGE_ENABLED: z
-    .string()
-    .default("false")
-    .transform(transformToBoolean("SWARM_STORAGE_ENABLED")),
+import {
+  z,
+  chainIdSchema,
+  booleanSchema,
+  createEnvSchema,
+} from "@blobscan/zod";
+
+const envSchema = createEnvSchema({
+  BEE_DEBUG_ENDPOINT: { schema: z.string().url(), optional: true },
+  BEE_ENDPOINT: { schema: z.string().url(), optional: true },
```

Comment:
Remember to check that either BEE_ENDPOINT or BEE_DEBUG_ENDPOINT are defined when SWARM_STORAGE_ENABLED=true as we spoke

Reply:
Yes, this was already [done](https://github.com/Blobscan/blobscan/blob/b2b2d10ac8ce6ed5104ffd4a257b319a7f2d90f6/packages/blob-storage-manager/src/storages/SwarmStorage.ts#L84) on previous PR as we discussed

Reply:
Oops thanks! I forgot it.

I have opened another PR fixing the logic. https://github.com/Blobscan/blobscan/pull/125